prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -361,7 +365,7 @@ class ActionHistory(
         // check that we're dealing with an external file
         val clientSource = report.sources.firstOrNull { it is ClientSource }
         if (clientSource != null) {
-            when (report.schema.topic.lowercase()) {
+            when (report.schema.topic.json_val.lowercase()) {
```

Comment:
This should be using the enum instead of json_val and lowercasing it.

Reply:
Good call, updated in latest commit.